src/app/api/events/route.ts

```diff
@@ -9,17 +9,38 @@ export async function GET(request: Request) {
 
   const url = new URL(request.url)
   const creatorId = url.searchParams.get('creatorId')
+  const eventId = url.searchParams.get('eventId')
 
-  const { data, error } = await supabase
-    .from('events')
-    .select()
-    .eq('creator', creatorId)
+  console.log(creatorId)
+  console.log(eventId)
 
-  if (error) {
-    return NextResponse.json({
-      status: 400,
-      message: 'Error fetching events',
-    })
+  if (creatorId !== null) {
+    console.log('in creatorId')
+
+    const { data, error } = await supabase
+      .from('events')
+      .select()
+      .eq('creator', creatorId)
+
+    if (error) {
+      return NextResponse.json({
```

Comment:
Please do a follow up PR but let's get a visible error page that explains the issue. Like "unknown user" here and "unknown event" below.

Reply:
Not in the API but as a separate page.